Tests/test_pile_ui.gd

```diff
@@ -0,0 +1,43 @@
+extends GutTest
+#test for ui pop up
+
+var _card_container_scene: PackedScene = load("res://Cards/CardContainer.tscn")
+var _card_container_scroll_scene: PackedScene  = load("res://#Scenes/CardScrollUI.tscn")
+var _card_container: CardContainer = null
+var _card_scroll: Control = null
+
+const SCROLL_CONTAINER = 2
+const GRID_CONTAINER = 0
+
+
+func before_all():
```

Comment:
should add an `after_all` to free the queue. Maybe even doing a `before_each` and `after_each` would be better in case we add more tests that would need a clean environment between each of them